.gitignore

```diff
@@ -1 +1,17 @@
-/tmp
\ No newline at end of file
+# System
+.DS_Store
+
+# Integrated Development Environment
+.idea
+.vscode
+
+# Package Manager
+## Cargo
+target
+
+# Build Data
+build
+
+# WASMS
```

Comment:
Why ignore the built wasm blob and digests? That's what we need to publish in the release.

Reply:
Ignore the local generated.
We will eventually use this script in CI.